app/Http/Controllers/RoomAPIController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Room;
+use Illuminate\Http\Request;
+
+class RoomAPIController extends Controller
+{
+   public function find(int $id): \Illuminate\Http\JsonResponse
+   {
+       $room = Room::find($id);
```

Comment:
Here you should really be using with() to load the type straight away - like what was done in story 1